packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1760,7 +1745,11 @@ class DropdownButtonFormField<T> extends FormField<T> {
                     menuMaxHeight: menuMaxHeight,
                     enableFeedback: enableFeedback,
                     alignment: alignment,
-                    borderRadius: borderRadius ?? effectiveBorderRadius(),
+                    borderRadius: borderRadius ?? switch (resolvedBorder) {
+                      final OutlineInputBorder border   => border.borderRadius,
+                      final UnderlineInputBorder border => border.borderRadius,
+                      _ => null,
```

Comment:
I wonder if these have crept in to the code base, do you know @nate-thegrate?

Referencing https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-anonymous-parameter-names again.

Reply:
Responded in [other thread](https://github.com/flutter/flutter/pull/154753#discussion_r1783653646) 👍